zomes/holo_acorn/code/src/anchor/mod.rs

```diff
@@ -0,0 +1,126 @@
+extern crate hdk;
+extern crate hdk_proc_macros;
+extern crate holochain_json_derive;
+extern crate serde;
+extern crate serde_derive;
+extern crate serde_json;
+use hdk::entry_definition::ValidatingEntryType;
+use hdk::holochain_core_types::{
+    // agent::AgentId, dna::entry_types::Sharing, entry::Entry, link::LinkMatch,
+    dna::entry_types::Sharing,
+    entry::Entry,
+};
+
+pub fn init() {
+    let goals_anchor_entry = Entry::App(
+        "anchor".into(), // app entry type
+        // app entry value. We'll use the value to specify what this anchor is for
+        "goals".into(),
+    );
+    let edges_anchor_entry = Entry::App(
+        "anchor".into(), // app entry type
+        // app entry value. We'll use the value to specify what this anchor is for
+        "edges".into(),
+    );
+    let goal_members_anchor_entry = Entry::App(
+        "anchor".into(), // app entry type
+        // app entry value. We'll use the value to specify what this anchor is for
+        "goal_members".into(),
+    );
+    let goal_vote_anchor_entry = Entry::App(
+        "anchor".into(), // app entry type
+        // app entry value. We'll use the value to specify what this anchor is for
+        "goal_votes".into(),
+    );
+    let goal_comment_anchor_entry = Entry::App(
+        "anchor".into(), // app entry type
+        // app entry value. We'll use the value to specify what this anchor is for
+        "goal_comments".into(),
+    );
+    let agents_anchor_entry = Entry::App(
+        "anchor".into(), // app entry type
+        // app entry value. We'll use the value to specify what this anchor is for
+        "agents".into(),
+    );
+    let _ = hdk::commit_entry(&goal_comment_anchor_entry);
```

Comment:
These should then use `?` to return the Err as the Result if any of these calls fails for some reason

Reply:
threw the wrong "?"

Reply:
![image](https://user-images.githubusercontent.com/16279205/72155100-b72d7580-3388-11ea-9362-84151de3201d.png)
I'm making this mistake

Reply:
Are you stuck on it?

Reply:
yes, I am. can you help me? I wrote you on mattermost